frontend/src/src/components/ReviewPage/index.js

```diff
@@ -81,7 +84,7 @@ export default function ReviewPage(props) {
         })
     }, [petitioner, petitions])
 
-    function postGeneratorRequest(petitioner, petition, index) {
+    async function postGeneratorRequest(petitioner, petition, index) {
```

Comment:
style nit: we don't use the function keyword in most places, just arrow functions

Reply:
I always forget, they handle `this` binding differently--my rule is use arrow operators until they aren't doing what you expect, then try function